casper/src/main/resources/PoS.rhox

```diff
@@ -37,20 +38,22 @@ new  PoS,
     posId(`rho:rchain:deployerId`),
     PendingRewards, MergeIntMaps, treeHashMapCh,
     sysAuthTokenOps(`sys:authToken:ops`),
+    multiSigRevVaultCh, coopWalletCh,
```

Comment:
"Vault" instead of "Wallet" here and below.  In all other blockchains, "wallets" hold keys and decrypt them when you type in your password, so we're trying to move away from that terminology to describe on-chain contracts.

Reply:
Thanks for the lesson! I changed the terminology.